scripts/test/deployAll/system-test-instance.properties

```diff
@@ -65,193 +65,23 @@ sleeper.optional.stacks=SystemTestStack,CompactionStack,GarbageCollectorStack,In
-
-# The timeout for the lambda that identifies partitions that need splitting in seconds
-sleeper.partition.splitting.finder.timeout.seconds=900
-
-# The amount of memory for the lambda that splits partitions
-sleeper.partition.splitting.memory=2048
-
-# The timeout for the lambda that splits partitions in seconds
-sleeper.partition.splitting.timeout.seconds=900
-
-# The amount of memory in MB for the lambda that executes queries
-sleeper.query.processor.memory=2048
-
-# The timeout for the lambda that executes queries in seconds
-sleeper.query.processor.timeout.seconds=900
-
-# The maximum number of simultaneous connections to S3 from a single client
-sleeper.s3.max-connections=25
-
-# The frequency with which the query processing lambda refreshes its knowledge of the system state
-# (i.e. the partitions and the mapping from partition to files), in seconds.
-sleeper.query.processor.state.refresh.period.seconds=60
-
-# The maximum number of records to include in a batch of query results send to
-# the results queue from the query processing lambda.
-sleeper.query.processor.results.batch.size=2000
-
-# Fargate version
-sleeper.fargate.version=1.4.0
-
-# The EC2 instance type to use for compaction tasks (when not using Fargate).
-sleeper.compaction.ec2.type=t3.xlarge
-
-# The minimum / initial desired / maximum number of instances for the cluster.
-sleeper.compaction.ec2.pool.minimum=0
-sleeper.compaction.ec2.pool.desired=0
 # Maximum number of EC2s limited for system test
 sleeper.compaction.ec2.pool.maximum=20
 
-# What launch type should compaction containers use? Valid options: FARGATE, EC2
-sleeper.compaction.ecs.launch.type=FARGATE
-
-# The size in GiB of the root EBS volume attached to the EC2 instances.
-sleeper.compaction.ec2.root.size=50
-
-# The frequency with which the WriteData class refreshes the list of partitions from the state store
-# (NB Refreshes only happen once a batch of data has been written so this is a lower bound
-# on the refresh frequency.)
-sleeper.ingest.partition.refresh.period=120
-
-# The frequency with which the lambda that checks whether ingest tasks need creating runs (in minutes).
-sleeper.ingest.task.creation.period.minutes=1
-
-# The number of files to read in a compaction job. Note that the state store
-# must support atomic updates for this many files. For the DynamoDBStateStore this
-# is 11.
-# (NB This does not apply to splitting jobs which will run even if there is only 1 file.)
-sleeper.compaction.files.batch.size=11
-
-# The way in which records are held in memory before they are written to a local store
-# Valid values are 'arraylist' and 'arrow'
-sleeper.ingest.record.batch.type=arrow
-
 # The way in which partition files are written to the main Sleeper store
 # Valid values are 'direct' (which writes using the s3a Hadoop file system) and 'async' (which writes locally and then
 # copies the completed Parquet file asynchronously into S3)
 sleeper.ingest.partition.file.writer.type=async
```

Comment:
Should this be made the default? It seems like we should be testing against the default, but we've used this value for some time.

Reply:
Yes, we can delete this.